pkg/installation/install.go

```diff
@@ -41,7 +41,7 @@ const (
 	krewPluginName = "krew"
 )
 
-func downloadAndMove(version, uri string, fos []index.FileOperation, downloadPath, installPath, forceDownloadFile string) (dst string, err error) {
+func downloadAndMove(version, sha256sum, uri string, fos []index.FileOperation, downloadPath, installPath, forceDownloadFile string) (dst string, err error) {
```

Comment:
7 arguments? I hope this is a very very private function. Can you add a todo so we get reminded that this needs to be refactored?

Reply:
The TODO is already at `getDownloadTarget` (see that method's diff). Once that's fixed we'd be just passing the returned struct here to simplify.